packages/core/src/config.js

```diff
@@ -148,6 +148,7 @@ export const snapshotSchema = {
       }
     },
     exec: {
+      error: 'must be a function, function body, or array',
```

Comment:
> must be a function, function body, or array of functions

? 🤷🏼‍♂️ Take it or leave it

Reply:
I had much more of a mouthful at first

> must be a function, function body, or array of functions or function bodies

I like your compromise

Reply:
Yo dawg, I heard you like functions so we put a function body inside of a function body so you can evaluate your functions